script/fetch_database.sh

```diff
@@ -0,0 +1,13 @@
+set -e
```

Comment:
Not sure what this script is supposed to do. Where does feature.raw come from?

Would it make sense to have deploy_master.sh and deploy_slave.sh?

BTW, once we have @hubifant's script, I think this will change.